electron/src/components/privilege-checker/privilege-checker.component.ts

```diff
@@ -23,7 +24,19 @@ import { UhkDeviceService } from '../../services/uhk-device.service';
 })
 export class PrivilegeCheckerComponent {
 
-    constructor(private router: Router, private uhkDevice: UhkDeviceService) {
+    private rootDir: string;
+
+    constructor(private router: Router,
+                private uhkDevice: UhkDeviceService,
+                @Inject(LOG_SERVICE) private logService: ILogService) {
+        if (isDev) {
+            this.rootDir = path.resolve(path.join(remote.process.cwd(), remote.process.argv[1]), '..');
+        }
```

Comment:
`}` should be in the same line as the `else` keyword according to our coding standards.